docs/3.0.0-beta.x/deployment/proxy.md

```diff
@@ -0,0 +1,68 @@
+# Proxying
```

Comment:
I think this part has to be in the `docs/3.0.0-beta.x/getting-started/deployment.md` file.
This is not a provider but just configurations for the server.